jc3mpserverconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "announce": {{announce}},
+  "description": "{{description}}",
+  "host": "0.0.0.0",
+  "httpPort": {{hport}},
+  "logLevel": 7,
+  "logo": "",
```

Comment:
Any reason to not include these two in the jc3mpconfig.json?

Reply:
Yes -- according to the maintainer, the log level doesn't do anything currently, and the logo feature is broken, and populating it will crash the server on launch.

Reply:
Lmao

Reply:
Lol my exact response too.